devtools/buildVisualStories.js

```diff
@@ -4,7 +4,7 @@ const path = require('path');
 const srcPath = path.join(__dirname, '../src');
 
 const isComponentDirectory = (source) => {
-    const ignoredDirectories = ['utils', 'Docs'];
+    const ignoredDirectories = ['utils', 'Docs', 'Identifier', 'Image', 'Panel'];
```

Comment:
I needed to make these changes to get storybook to run locally?

Reply:
That's because those components were deleted 😃  I'd remove that then make sure your local is matching master